### DISCUSSION TITLE
Update email automatically when it changes in social platform

### DISCUSSION THREAD

**Discussion by nmapx** [ory/kratos#3493] [Q&A] [answered]
I was wondering if it's possible to update email in Kratos automatically after it's changed in social platform.
Assuming that user logs in with social so the account is already linked.

**Comment by vinckr** [comment, marked as answer]
Hey @nmapx 
You are looking for this feature probably: https://github.com/ory/kratos/issues/1912

**Reply by nmapx** [reply]
I see an open PR mentioned in the issue above. Any idea if it's gonna be merged? It's hanging there for a while now.

**Reply by vinckr** [reply]
Hello @nmapx 

In general there are no timelines for PRs. 
It really depends on customer demand. So if an Ory customer needs this we can allocate developers to work on it and implement it. Right now there is still a substantial amount of work involved whenever a new feature is added.
So if that is a feature you would like to see supported in Ory on a timeline, best reach out to your account manager or support@ory.sh.
